proto/dymension/rollapp/genesis.proto

```diff
@@ -14,8 +14,8 @@ message GenesisState {
   Params params = 1 [(gogoproto.nullable) = false];
   repeated Rollapp rollappList = 2 [(gogoproto.nullable) = false];
   repeated StateInfo stateInfoList = 3 [(gogoproto.nullable) = false];
-  repeated StateInfoIndex latestStateInfoIndexList = 4 [(gogoproto.nullable) = false];
-  repeated StateInfoIndex latestFinalizedStateIndexList = 5 [(gogoproto.nullable) = false];
-  repeated BlockHeightToFinalizationQueue blockHeightToFinalizationQueueList = 6 [(gogoproto.nullable) = false];
+  //repeated StateInfoIndex latestStateInfoIndexList = 4 [(gogoproto.nullable) = false];
```

Comment:
what is the point of storing some state that it can be already retrieved from the state info? imo is not really necessaries for the genesis and can lead to inconsistencies and is better to just get it from state info.